.github/workflows/commitlint.yml

```diff
@@ -5,6 +5,27 @@ name: Lint Commit Messages
 on:
   - pull_request
 
+defaults:
+  run:
+    shell: bash
+
 jobs:
   commitlint:
-    uses: openedx/.github/.github/workflows/commitlint.yml@master
+    # uses: openedx/.github/.github/workflows/commitlint.yml@master
```

Comment:
This temporary workaround is required until the fix in https://github.com/openedx/.github/pull/109 is merged.